ghengin-core/ghengin-core-indep/Ghengin/Core/Shader.hs

```diff
@@ -111,3 +115,65 @@ instance KnownSymbol name => ShaderData (StructMat4 name) where
 
 instance KnownSymbol name => ShaderData (StructFloat name) where
   type FirType (StructFloat name) = FIR.Struct '[ name 'FIR.:-> Float ]
+  
+-- ** FIR Vector
+instance (KnownNat n, Block x, Storable x) => Block (V n x) where
+  type PackedSize (V n x) = n * (PackedSize x)
+
+  alignment140 _ = Store.alignment (undefined :: (V n x))
+  alignment430 = alignment140
+
+  sizeOf140 _ = Store.sizeOf (undefined :: (V n x))
+  sizeOf430 = sizeOf140
```

Comment:
I thought of adding an `error` here, but then I checked Geomancy and saw that it does the same thing. https://hackage-content.haskell.org/package/geomancy-0.3.0.0/docs/src/Geomancy.Vec3.html#local-6989586621679124795

I believe Geomancy is just using it as a placeholder for now until it's implemented. It shouldn't cause crashes or anything, right? Just affect performance if the engine actually used the 430 layout?

Thoughts @dpwiz?